### ISSUE TITLE
Multiplayer not included in Android release

### ISSUE THREAD

**Issue by TheDude53** [OpenRCT2/OpenRCT2#14150]
The multiplayer feature appears to be missing from the latest release for Android, 0.3.2.
#6143 mentions that OpenSSL/LibreSSL was included with Android, which would suggest it was enabled at one point.
Why was this feature removed? Are there plans for it to be re-enabled?

**Comment by Gymnasiast** [comment]
@janisozaur

**Comment by janisozaur** [comment]
Yes, it was enabled at one point, but libraries changed a bit since then and it was easier to disable that feature.

Android port is usually lower priority for us, as we think there's little demand, but we're open to having our opinions changed.

**Comment by TheDude53** [comment]
I think that since OpenRCT2's goal is to re-implement RCT2, it would be beneficial to polish up the Android release and match [what RCT did](https://play.google.com/store/apps/details?id=com.atari.mobile.rctclassic).

**Comment by tupaschoal** [comment]
Don't get us wrong, we want to have all versions matching, we just don't have the bandwidth to maintain them all. Android is specially difficult because it involves extra/different stuff when compared to all other releases.

**Comment by janisozaur** [comment]
Oh it sure would be nice, but we have finite manpower available and again, our priorities lie [elsewhere](https://github.com/OpenRCT2/OpenRCT2/projects/6)

**Comment by TheDude53** [comment]
Those are fair reasons and I agree, the finite manpower available should be working on preexisting priorities.

**Comment by Basssiiie** [comment]
Multiplayer has been enabled again on Android since #21631 (v0.4.10) so that should resolve this issue. 🙂 If there's any issues still, please leave a message and we can reopen it again.
